dali/operators/reader/loader/coco_loader.h

```diff
@@ -1,4 +1,4 @@
-// Copyright (c) 2019-2023, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
+// Copyright (c) 2019-2024, NVIDIA CORPORATION & AFFILIATES. All rights reserved.
```

Comment:
I don't see any changes here besides reformatting. Revert?

Reply:
There are some (removing using ImageIdPairs) and renaming of some member fields